pokecli.py

```diff
@@ -78,14 +81,19 @@ def init_config():
 def main():
     # log settings
     # log format
-    logging.basicConfig(level=logging.DEBUG, format='%(asctime)s [%(module)10s] [%(levelname)5s] %(message)s')
+    logging.basicConfig(level=logging.DEBUG, format='%(asctime)s [%(module)16s] [%(levelname)5s] %(message)s')
     # log level for http request class
     logging.getLogger("requests").setLevel(logging.WARNING)
     # log level for main pgoapi class
     logging.getLogger("pgoapi").setLevel(logging.INFO)
     # log level for internal pgoapi class
     logging.getLogger("rpc_api").setLevel(logging.INFO)
 
+    logging.getLogger("bot").setLevel(logging.INFO)
+    logging.getLogger("stepper").setLevel(logging.INFO)
+    logging.getLogger("seen_fort_worker").setLevel(logging.INFO)
+
```

Comment:
@berend I think these `setLevel` calls above should be put inside the `Bot` class to keep things isolated. Also you forgot the `PokemonCatchWorker` logger.